### PR TITLE
Blockquote without border and extra margin

### PR DESCRIPTION
**Pull request by e-marie-w**
Originally tracked https://github.com/uiowa/uids/issues/624

But functionality was moved to https://github.com/uiowa/uids/pull/626 so that we didnt have build problems.
This is still useful to correct for build problems, but not necessary now.

Not sure if we generated the `package-lock.json` file correctly, may need to review.

### CODE REVIEW

**Comment by e-marie-w** [comment]
@bspeare do you have any idea on what to do with the `package-lock.json` file, or is this normal?

**Comment by bspeare** [comment]
I wonder if we can slightly adjust the modifier classes to be something like:

`blockquote--remove-margin`
`blockquote--remove-stylization`

We will be introducing more blockquote styles in the future, so the `remove-stylization` will be more useful for that.

File: src/components/blockquote/blockquote.scss [hunk 1]
```diff
@@ -275,4 +282,26 @@ blockquote {
...
+
+  &.blockquote--no-border {
+    border-width: 0;
+
+    &:before {
+      height: 0;
+    }
+
+    .blockquote__paragraph {
+      &:after {
```
[the remark this answers is not in the document]
Comment (GaryRidgway) [reply src/components/blockquote/blockquote.scss:294]: Looks like [ultimately it doesn't matter,](https://css-tricks.com/to-double-colon-or-not-do-double-colon/) but double it probably the way to go in the long run?

**Comment by bspeare** [comment]
We should probably move all the `package` and `gulp` updates to a separate pr.  I think we will need more time to test those and fix them.

**Comment by e-marie-w** [comment]
> We should probably move all the `package` and `gulp` updates to a separate pr. I think we will need more time to test those and fix them.

I like that plan! It looks like there's PRs to update the dependencies already anyways.

**Pull request closed without merging**